oss.gg/3_follow_the_unkey_x_account.md

```
@@ -25,4 +25,33 @@ Your turn 👇
+
+» 11-October-2024 by [Aritra Sadhukhan](@AritraDevelops)
+» Link to Account: https://x.com/AritraDevelops
+
+» 11-October-2024 by [Emily M. Ahtunan](@shemmariea)
+» Link to Account: https://x.com/shemmariea
+
+» 11-October-2024 by [Devang Rakholiya](@Devang0907)
+» Link to Account: https://x.com/Devang0907
+
+» 13-October-2024 by [Atharva Deshmukh](@atharva-3000)
+» Link to Account: https://x.com/0x_atharva
```

Markdownlint

Bare URL used

(MD034, no-bare-urls)